cluster/swarm/cluster.go

```diff
@@ -330,6 +330,7 @@ func (c *Cluster) validatePendingEngine(engine *cluster.Engine) bool {
 	// set engine state to healthy, and start refresh loop
 	engine.ValidationComplete()
 	c.engines[engine.ID] = engine
+	engine.AfterEngineValidationComplete()
```

Comment:
@dongluochen 
Thank you for reviewing , I think you'r right. Have changed code.